code/modules/projectiles/guns/ballistic/pistol.dm

```diff
@@ -80,6 +80,7 @@
 	name = "flat gun"
 	desc = "A 2 dimensional gun.. what?"
 	icon_state = "flatgun"
+	can_suppress = FALSE
```

Comment:
Flat guns down have sprites by the by for suppress, also the stick man fire .45 but I dont want to change that here